DayrSuryan/DSEthiop10.xml

```diff
@@ -481,10 +488,23 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                </physDesc>
 
                <history>
-               <!--   <origin>
-                  </origin>-->
+                   <origin>
+                <origDate notBefore="1930" notAfter="1960">From 1930 to the mid-20th century.</origDate>
+              <persName ref="PRS5226haylase">King (<foreign xml:lang="gez">nǝguśǝna</foreign>) Ḫāyla Śǝllāse I</persName> is mentioned in <locus target="#30r"/>.
+                <note>The manuscript was copied and bound in Egypt. Several scribes participated in writing, unnamed <ref target="#h2"/> and <ref target="#h3"/>,
```

Comment:
No need for `<note>`, the text can be freely written in origin.

Reply:
Ok.